src/app/multiple-records/multiple-records.container.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit, ChangeDetectionStrategy } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Store, select } from '@ngrx/store';
+import { IAppState } from '@app/store/state/app.state';
+
+import { selectVehicleTechRecordModelHavingStatusAll } from '@app/store/selectors/VehicleTechRecordModel.selectors';
+import { VehicleTechRecordModel } from '@app/models/vehicle-tech-record.model';
+
+@Component({
+  selector: 'vtm-multiple-records-container',
+  template: `
+    <ng-container *ngIf="techRecords$ | async as techRecords">
```

Comment:
this is not techRecords. It is vehicleTechRecords at this stage. Lets be semantic to lessen confusion